### ISSUE TITLE
User lost username and couldn't log in

### ISSUE THREAD

**Issue by aankit** [nycdoe-cs4all/blueprint-learning-site#40]
User tried to log in and got login or password incorrect error

They tried to reset password but still got login or password incorrect error

They emailed me and I told them that their user name was not what they expected.

**Comment by antiboredom** [comment]
What if I add a line in the reset email that says what their username is?

**Comment by antiboredom** [comment]
alternatively we could get rid of usernames all together and just use email for login. Thoughts?

**Comment by aankit** [comment]
yes, I'm all for email. many users have been putting in their email when
trying to login
